Makefile

```diff
@@ -14,9 +14,10 @@
 
 # Image repo/tag to use all building/pushing image targets
 DOCKER_REGISTRY ?= ghcr.io/kubestellar/kubestellar
-IMAGE_TAG ?= 0.20.0-alpha.1
 CMD_NAME ?= controller-manager
-IMG ?= ${DOCKER_REGISTRY}/${CMD_NAME}:${IMAGE_TAG}
+IMAGE_TAG ?= $(shell git rev-parse --short HEAD) 
+CORE_IMAGE_REPO ?= ${DOCKER_REGISTRY}/${CMD_NAME}
```

Comment:
FYI: "CORE" is not the most helpful qualifier here. We defined "core" as the new word for "center", when we generalized from edge computing to general multi-cluster. There are several container images that are relevant to the core/center. I would name this something more specific, like "KCM_IMAGE_REPO" ("KCM" for Kubestellar Controller Manager).

Reply:
Changed CORE_IMAGE_REPO to IMAGE_REPO.